### PR TITLE
connect wallet v2

### PR DESCRIPTION
**Pull request by MananTank**

### CODE REVIEW

File: components/app-layouts/providers.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export const DashboardThirdwebProvider: ComponentWithChildren<
 
   return (
     <ThirdwebProvider
-      queryClient={queryClient}
+      // queryClient={queryClient}
```
Comment (jnsdls) [inline components/app-layouts/providers.tsx:82]: we definitely can't lose passing this this

File: components/contract-components/contract-deploy-form/custom-contract.tsx [hunk 2]
```diff
@@ -91,8 +91,8 @@ const CustomContractForm: React.FC<CustomContractFormProps> = ({
     defaultValues: {
       addToDashboard: true,
       deployParams: deployParams.reduce((acc, param) => {
-        acc[param.name] = replaceTemplateValues(
-          fullPublishMetadata.data?.constructorParams?.[param.name]
+        acc[param.name || ""] = replaceTemplateValues(
```
Comment (jnsdls) [inline components/contract-components/contract-deploy-form/custom-contract.tsx:94]: @joaquim-verges I thought we fixed the param.name always being there?

Comment (joaquim-verges) [reply components/contract-components/contract-deploy-form/custom-contract.tsx:94]: needs to be schema.parsed i think

Comment (jnsdls) [reply components/contract-components/contract-deploy-form/custom-contract.tsx:94]: doesn't work either

Comment (jnsdls) [reply components/contract-components/contract-deploy-form/custom-contract.tsx]: fixed via various PRs to the sdk

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/auth**](https://thirdweb-dlf00z37l.thirdweb-preview.com/auth) | [**/dashboards**](https://thirdweb-dlf00z37l.thirdweb-preview.com/dashboards) |
|:-:|:-:|
|<a href="https://thirdweb-dlf00z37l.thirdweb-preview.com/auth">Screenshot of /auth</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/JJn2Ox1oJPn7HTSi-xVe-orQKyYx-iLyFOO_Q9nPKbkc35wWafLdTvNWtyVoYAIGMw_NawieAbgYJBuxLy7PpA.png">(view full size)</a> |<a href="https://thirdweb-dlf00z37l.thirdweb-preview.com/dashboards">Screenshot of /dashboards</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/L6dVPhD8kj1B5LB6JWIb1Vrjb3P4Uh-jEmixniJdUE_DKBDoWz-ILj3-OHSfYib9NRu6Z-F-r3rYTsv-jOPlXQ.png">(view full size)</a> |



| [**/deploy**](https://thirdweb-dlf00z37l.thirdweb-preview.com/deploy) | [**/solutions/commerce**](https://thirdweb-dlf00z37l.thirdweb-preview.com/solutions/commerce) |
|:-:|:-:|
|<a href="https://thirdweb-dlf00z37l.thirdweb-preview.com/deploy">Screenshot of /deploy</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/UVrn4SWxj0pZVP-WfCb1T0qxCbOgl0ptDWOZIz-sehIdAak7LO5U66gtFdBYwk7VHsnG6ddIhloos9KnF6EfqQ.png">(view full size)</a> |<a href="https://thirdweb-dlf00z37l.thirdweb-preview.com/solutions/commerce">Screenshot of /solutions/commerce</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/QTgJJFsmE5RzpewbANDT4z9n3Y8kkUfudwkLwaOJ6t0y0AusX2X4yjtD2sb2uGLTib1WcbD7Qo2WFYqPlE3baw.png">(view full size)</a> |



| [**/solutions/gaming**](https://thirdweb-dlf00z37l.thirdweb-preview.com/solutions/gaming) |
|:-:|
|<a href="https://thirdweb-dlf00z37l.thirdweb-preview.com/solutions/gaming">Screenshot of /solutions/gaming</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/_w_x3A8IbUZSd2F1OGPhGexEcWfBfW8xaSZu2SYJGu0zPoiu_TAcnCCsKoD7U1SwpeWAgHd0Wb0rl50JwLrh6Q.png">(view full size)</a> |







Commit f3e7da3f7259f9920d5ce7feddd8ec22da146982 (https://thirdweb-dlf00z37l.thirdweb-preview.com).

**Pull request merged**